### PR TITLE
Remove From Course and Remove From Event.

### PR DESCRIPTION
**Pull request by Fourstarfish**
Implemented Remove From Course and Remove From Event. These have not been tested yet. The views have not been linked to other views yet either.

### CODE REVIEW

File: src/use_case/RemoveFromCourseInteractor.java
Comment (alexgphan) [inline src/use_case/RemoveFromCourseInteractor.java:1]: When you remove course from employee it automatically removes that employee from course.staff according to the removeCourse method. We might not need to include the line curCourse.removeStaff(curEmployee).

File: src/use_case/RemoveFromEventInteractor.java
Comment (alexgphan) [inline src/use_case/RemoveFromEventInteractor.java:1]: Also I'm not 100% sure but does prepare view show each error message or just the first error that appears?

**Review by alexgphan — COMMENTED** [review]
Don't feel fully comfortable reviewing the presenter and the views but when looking at the other files it seems good. I'm not sure if they want us to include docstring for the public methods inside the usecases and adapters but we can just ask on piazza real quick.

File: src/use_case/RemoveFromEventInteractor.java
Comment (xubosen) [reply src/use_case/RemoveFromEventInteractor.java:1]: Currently it shows the first error-message that occurs.

**Review by xubosen — COMMENTED** [review]
On second review, everything seems to be in place, except for the one comment Alex brought up, which is redundant code.

Other than that, I think we need documentation for our public classes and methods, so that is something that should be done next. 

Unit testing is also highly recommended since more complicated cases of removing may need to be tested before we assert that the code is working.

**Review by xubosen — APPROVED** [review]
Problems fixed. Testing still needed, but coming soon according to discussion in person.

**Pull request merged**